static/js/controllers/edit-contact.js

```diff
@@ -86,6 +92,10 @@ var libphonenumber = require('libphonenumber/utils'),
           };
           $scope.category = $scope.contact.type === 'person' ? 'person' : 'place';
           $scope.contactId = null;
+          if ($scope.category === 'place') {
+            // set the Primary Contact field for places to be 'blank'
+            $scope.contact.contact = $scope.parents[0];
```

Comment:
Won't this get persisted? We might need a check in the save method that `if (contact.contact && !contact.contact._id) delete contact.contact;` ?